core/chains/evm/config/toml/defaults/BSC_Mainnet.toml

```diff
@@ -10,6 +10,8 @@ NoNewFinalizedHeadsThreshold = '45s'
 
 [GasEstimator]
 PriceDefault = '5 gwei'
+# Can change edit BSC node config's GasPrice under Eth.Miner from 3000000000 to 1000000000 and change PriceMin to 1 gwei to enable cheaper transactions
```

Comment:
We usually don't want NOPs to change configs without us asking them to. It's led to issues in the past. I think it's still important noting why this value is chosen though. Could you reword this comment? Maybe something like:
```suggestion
# Set to the BSC node's default Eth.Miner.GasPrice config
```

Reply:
After this fix goes out we were going to let NOPs know they should overwrite this config back down to 1gwei and also change their BSC node config to match ([slack thread](https://chainlink-core.slack.com/archives/C041J282GE9/p1723740413153899?thread_ts=1719951097.450339&cid=C041J282GE9))

Reply:
It just seems like we added the instructions to the toml instead of just using the comments are a description of the config. This info will be conveyed to them when you reach out so seems ok to omit here.

Reply:
sounds good. will commit the suggestion